node/cn/handler.go

```diff
@@ -122,6 +122,8 @@ type ProtocolManager struct {
 
 	nodetype          common.ConnType
 	txResendUseLegacy bool
+
+	tmpstop bool
```

Comment:
The `tmpstop` is a little bit confusing to me to figure out what it does. What about changing to the more specific name?

Reply:
Thx. I'll reflect.